src/main/scala/ai/eto/rikai/sql/model/Model.scala

```diff
@@ -68,10 +70,18 @@ class ModelNameException(message: String) extends Exception(message);
   * @param uri the model uri.
   * @param funcName the name of a UDF which will be called when this model is invoked.
   */
-class SparkUDFModel(val name: String, val uri: String, val funcName: String)
-    extends Model
+class SparkUDFModel(
+    val name: String,
+    val uri: String,
```

Comment:
i forgot is this is the artifact or the spec uri

Reply:
This is the URI specified from `CREATE TABLE using URI`?